### PR TITLE
Rename the backend cargo features

### PR DESCRIPTION
**Pull request by ogoffart**

### CODE REVIEW

File: internal/interpreter/Cargo.toml [hunk 1]
```diff
@@ -40,44 +40,40 @@ std = ["i-slint-core/std"]
...
 
-## This feature is an alias for `backend-qt`
-eventloop-qt = ["backend-qt"]
+## Enable the `winit` backend and make it capable of renderer using the
+## [femtovg](https://crates.io/crates/femtovg) crate.
+renderer-winit-femtovg = ["i-slint-backend-selector/renderer-winit-femtovg", "std"]
 
-## The `femtovg` crate is available for rendering.
-renderer-femtovg = ["i-slint-backend-selector/renderer-femtovg", "std"]
+## (Experimental) Enable the `winit` backend and make it capable of renderer using [Skia](https://skia.org/)
```
Comment (hunger) [inline internal/interpreter/Cargo.toml:68]: typo: of rendering

File: internal/interpreter/Cargo.toml [hunk 2]
```diff
@@ -40,44 +40,40 @@ std = ["i-slint-core/std"]
...
+renderer-winit-skia = ["i-slint-backend-selector/renderer-winit-skia", "std"]
 
-## The Skia based rendering engine.
-renderer-skia = ["i-slint-backend-selector/renderer-skia", "std"]
+## (Experimental) Same as `renderer-winit-skia`, but Skia will always use OpenGL.
```
Comment (hunger) [inline internal/interpreter/Cargo.toml:71]: Does this include renderer-winit-skia and turns on an extra knob or does this exclude renderer-winit-skia, replacing it with a special version?

File: internal/interpreter/Cargo.toml [hunk 3]
```diff
@@ -40,44 +40,40 @@ std = ["i-slint-core/std"]
...
+renderer-winit-skia-opengl = ["i-slint-backend-selector/renderer-winit-skia-opengl", "std"]
 
-## The Skia based rendering engine that will always use OpenGL.
-renderer-skia-opengl = ["i-slint-backend-selector/renderer-skia-opengl", "std"]
+## (Experimental) Enable the `winit` backend and make it capable of renderer using the software renderer
```
Comment (hunger) [inline internal/interpreter/Cargo.toml:74]: typo: rendering

File: internal/interpreter/Cargo.toml (hunk 1 above)
Comment (ogoffart) [reply internal/interpreter/Cargo.toml:68]:
```suggestion
## (Experimental) Enable the `winit` backend and make it capable of rendering using [Skia](https://skia.org/)
```

File: internal/interpreter/Cargo.toml [hunk 4]
```diff
@@ -40,44 +40,40 @@ std = ["i-slint-core/std"]
 
 #! ### Backends
 
+#! See the documentation of the [`slint` crate](https://docs.rs/slint/latest/slint/#backends)
+
 ## The Qt backend feature uses Qt for the windowing system integration and rendering.
-## This backend is required to use the `native` style.
+## This backend also provide the `native` style.
```
[the remark this answers is not in the document]
Comment (ogoffart) [reply internal/interpreter/Cargo.toml:46]:
```suggestion
## This backend also provides the `native` style.
```

File: api/rs/slint/Cargo.toml [hunk 5]
```diff
@@ -58,54 +58,58 @@ unsafe-single-threaded = ["i-slint-core/unsafe-single-threaded"]
...
+#!  * `SLINT_BACKEND=winit` forces the winit backend
+#!  * `SLINT_BACKEND=winit-femtovg` forces the winit backend with the femtovg renderer
+#!  * `SLINT_BACKEND=winit-skia` forces the winit backend with the skia renderer
+#!  * `SLINT_BACKEND=winit-software` forces the winit backend with the software renderer
+#! If the specified backend is not existing, the default will be used.
+
+
 ## The Qt backend feature uses Qt for the windowing system integration and rendering.
-## This backend is required to use the `native` style.
+## This backend also provide the `native` style.
```
[the remark this answers is not in the document]
Comment (ogoffart) [reply api/rs/slint/Cargo.toml:83]:
```suggestion
## This backend also provides the `native` style.
```

File: internal/interpreter/Cargo.toml (hunk 3 above)
Comment (ogoffart) [reply internal/interpreter/Cargo.toml:74]:
```suggestion
## (Experimental) Enable the `winit` backend and make it capable of rendering using the software renderer
```

**Pull request merged**